### ISSUE TITLE
plans to improve

### ISSUE THREAD

**Issue by ricmetal** [joaopsys/AutoChronoGG#4]
any plans to improve the script? 

1 - auto clicking the bonus reward chests?
2 - or at least returning a message when there's a chest reward to click?

**Comment by SavageCore** [comment]
1. Not sure
2. [Possible](https://github.com/telekineticyeti/lazychrono/blob/a7d57afc435640877a1f895ce6e79658868726b0/app.js#L26)
3. Create multiple cron jobs specifying your different JWT for each, the cookie saving mechanism will be defunct

**Comment by joaopsys** [comment]
I just checked my ChronoGG transaction history and the bonus chests (3-day bonus chest, 7-day, etc) seem to be activated automatically, without having to click on them. Is that what you are referring to? Or what chests are we talking about?

**Comment by SavageCore** [comment]
Yeah, 1 seems unnecessary. I have code ready for 2, can send PR once #5 lands. I just replicated the output of `lazychrono` linked above.
